graql/reasoner/query/ReasonerQueryFactory.java

```diff
@@ -60,12 +60,14 @@ public ReasonerQueryFactory(ConceptManager conceptManager, QueryCache queryCache
         this.ctx = new ReasoningContext(this, conceptManager, queryCache, ruleCache, statistics);
     }
 
-    public void enableInferTypes() {
+    public ReasonerQueryFactory enableInferTypes() {
```

Comment:
not sure i like this change, its not a builder really, or returning a new instance ? Not having the "fluid" pattern here is fine imo